src/utils/backend.js

```diff
@@ -43,11 +44,22 @@ export default class Backend {
 
   static getProfile = async (address) => backendFetch(`profile/${address}`);
 
-  static sendProfileData = async (address, postParam) => backendFetch(`profile/${address}`, {
-    method: 'post',
-    body: JSON.stringify(postParam),
-    headers: { 'Content-Type': 'application/json' },
-  });
+  static sendProfileData = async (address, postParam) => {
+    const request = {
+      method: 'post',
+      body:
+        postParam instanceof FormData ? postParam : JSON.stringify(postParam),
+    };
+
+    Object.assign(
+      request,
+      !(postParam instanceof FormData) && {
+        headers: { 'Content-Type': 'application/json' },
+      },
+    );
```

Comment:
Here we assign the `headers` prop only in the case where no image was uploaded. 

We can optimize if we find a more clean way to have a null-safe object property assign.

Reply:
I'm proposing to do it in a way as it was before:
```js
static sendProfileData = async (address, postParam) => backendFetch(`profile/${address}`, {
  method: 'post',
  body: postParam instanceof FormData ? postParam : JSON.stringify(postParam),
  ...!(postParam instanceof FormData) && { headers: { 'Content-Type': 'application/json' } },
});
```